hw/ip/otbn/data/otbn.ld.tpl

```diff
@@ -22,15 +22,24 @@ MEMORY
 
 SECTIONS
 {
-    .text ALIGN(4) :
+    .text ORIGIN(imem) : ALIGN(4)
     {
         *(.text*)
-    } >imem AT >imem_load
 
-    .data ALIGN(32) :
+        /* Align section end. Shouldn't really matter, but might make binary
+           blobs a bit easier to work with. */
+        . = ALIGN(32);
```

Comment:
This should probably just be 4-byte aligned, given it's code. The `.data` section I could understand aligning the end to 32-bytes, given the start is so aligned too.
```suggestion
        . = ALIGN(4);
```

Reply:
You're completely right, thanks! This was a copy-paste error from code that I later deleted! I've kept the data section 4-byte aligned because I think it's really about the thing that does the loading and Ibex has a 32-bit bus to load up the contents of both IMEM and DMEM.